### PR TITLE
Compute Studio

### PR DESCRIPTION
**Pull request by jdebacker**
This PR adds the necessary files for creating a web app on [Compute.Studio](https://compute.studio).

### CODE REVIEW

File: compconfig/compconfig/functions.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    '''
+    plt = op.ss_profiles(
+        base_ss, base_params, reform_ss=reform_ss,
+        reform_params=reform_params, by_j=True, var=var,
+        plot_title='Labor Supply in Baseline and Reform Policy')
+    comp_dict = {
+        "model_version": ogusa.__version__,
+        "renderable": [
+            {
+              "media_type": "matplotlib",
```
Comment (jdebacker) [inline compconfig/compconfig/functions.py]: @hdoupe I couldn't find an example of using a matplotlib figure.  Can you suggest how this might be done?

**Comment by hdoupe** [comment]
@jdebacker PR #495 looks good! I made a couple updates to the files to meet the updated criteria and opened a PR on this branch here: https://github.com/jdebacker/dynamic/pull/13

File: compconfig/compconfig/functions.py (hunk 1 above)
Comment (hdoupe) [reply compconfig/compconfig/functions.py]: @jdebacker Is it interactive? If not, you can dump it to a PNG file. I'll add an example to the docs showing how to do that.

Comment (jdebacker) [reply compconfig/compconfig/functions.py]: @hdoupe It is not interactive - I just want a static png file.  Thanks!

**Comment by rickecon** [comment]
I love this and am very excited.

File: compconfig/compconfig/functions.py (hunk 1 above)
Comment (hdoupe) [reply compconfig/compconfig/functions.py]: @jdebacker here's an example for matplotlib results: https://docs.compute.studio/publish/outputs/#pngjpeg

**Comment by rickecon** [comment]
@jdebacker . As you know, the error is the `cs-config/cs_config/tests/test_functions.py` file failing when it tries to `from cs_kit import CoreTestFunctions`. The Compute.Studio [PR](https://github.com/compute-tooling/compute-studio-kit/pull/3) you cite above says that you have to install `cs_kit`. Should we add `cs_kit` to the `environment.yml` and to the `requirements.txt`?

**Comment by jdebacker** [comment]
@rickecon Good call - I wasn't sure if `pytest` was going to pick up the `cs-config` tests, but I guess it does.   I added `cs_kit` to `environment.yml`.  I think `requirements.txt` is only used when builting the Sphinx docs and the Compute.Studio files are covered there.

**Comment by jdebacker** [comment]
@rickecon This PR is ready for your review.

**Pull request merged**